src/Hero/__snapshots__/Hero.spec.tsx.snap

```diff
@@ -0,0 +1,23 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Component: Hero should be defined 1`] = `
+<section
+  className="anchor-hero sc-brqgnP dYemBD"
+  color="#FFFFFF"
```

Comment:
You can prevent this by not spreading in all props, which is probably a good standard to adopt or by pulling out that prop by name to prevent it from being 'rested/spread' to the child component.